contracts/TokenPool.sol

```diff
@@ -0,0 +1,170 @@
+pragma solidity 0.4.18;
+
+import { ERC20Basic } from "zeppelin-solidity/contracts/token/ERC20Basic.sol";
+import { Ownable } from "zeppelin-solidity/contracts/ownership/Ownable.sol";
+
+
+/**
+ * @title Mintable token interface
+ * @author Wojciech Harzowski (https://github.com/harzo)
+ */
+contract Mintable is ERC20Basic {
+    function mint(address to, uint256 amount) public;
+}
+
+
+/**
+ * @title Token pools register
+ * @dev Allows to register multiple pools of token with lockup period
+ * @author Wojciech Harzowski (https://github.com/harzo)
+ */
+contract TokenPool is Ownable {
+
+    /**
+     * @dev Represents registered pool
+     */
+    struct Pool {
+        uint256 amount;
+        uint256 lockTimestamp;
+    }
+
+    /**
+     * @dev Address of mintable token instance
+     */
+    Mintable public token;
+
+    /**
+     * @dev Indicates available token amount for each pool
+     */
+    mapping (string => Pool) private poolRegister;
+
+    modifier onlyNotZero(uint256 amount) {
+        require(amount != 0);
+        _;
+    }
+
+    modifier onlySufficientAmount(string pool, uint256 amount) {
+        require(amount <= poolRegister[pool].amount);
+        _;
+    }
+
+    modifier onlyUnlocked(string pool) {
+        /* solhint-disable not-rely-on-time */
+        require(now > poolRegister[pool].lockTimestamp);
+        /* solhint-enable not-rely-on-time */
+        _;
+    }
+
+    modifier onlyUnique(string pool) {
+        require(poolRegister[pool].amount == 0);
+        _;
+    }
+
+    modifier onlyValid(address _address) {
+        require(_address != 0);
+        _;
+    }
+
+    function TokenPool(Mintable _token)
+        public
```

Comment:
Put `public` modifier in the same line. It is far from exceeding 120 columns as set in `.solhint`.